requirements/audio_test.txt

```diff
@@ -7,3 +7,6 @@ speechmetrics @ git+https://github.com/aliutkus/speechmetrics
 fast-bss-eval >=0.1.0, <0.1.5
 torch_complex <=0.4.3  # needed for fast-bss-eval
 srmrpy @ git+https://github.com/jfsantos/SRMRpy
+librosa
```

Comment:
Is it needed only for test or regular run?

Reply:
It is needed for regular run